http/build.gradle

```diff
@@ -8,11 +8,19 @@ dependencies {
 
     implementation libs.bundles.json
     implementation libs.bundles.logging
+    compileOnly libs.aws.secrets
```

Comment:
the implemetation dependency should be added by the client project if they wish to use this OAuth2HttpClient